agent/agent.go

```diff
@@ -68,26 +68,27 @@ const (
 )
 
 type Options struct {
-	Filesystem                   afero.Fs
-	LogDir                       string
-	TempDir                      string
-	ScriptDataDir                string
-	ExchangeToken                func(ctx context.Context) (string, error)
-	Client                       Client
-	ReconnectingPTYTimeout       time.Duration
-	EnvironmentVariables         map[string]string
-	Logger                       slog.Logger
-	IgnorePorts                  map[int]string
-	PortCacheDuration            time.Duration
-	SSHMaxTimeout                time.Duration
-	TailnetListenPort            uint16
-	Subsystems                   []codersdk.AgentSubsystem
-	PrometheusRegistry           *prometheus.Registry
-	ReportMetadataInterval       time.Duration
-	ServiceBannerRefreshInterval time.Duration
-	BlockFileTransfer            bool
-	Execer                       agentexec.Execer
-	ContainerLister              agentcontainers.Lister
+	Filesystem                    afero.Fs
+	LogDir                        string
+	TempDir                       string
+	ScriptDataDir                 string
+	ExchangeToken                 func(ctx context.Context) (string, error)
+	Client                        Client
+	ReconnectingPTYTimeout        time.Duration
+	EnvironmentVariables          map[string]string
+	Logger                        slog.Logger
+	IgnorePorts                   map[int]string
+	PortCacheDuration             time.Duration
+	SSHMaxTimeout                 time.Duration
+	TailnetListenPort             uint16
+	Subsystems                    []codersdk.AgentSubsystem
+	PrometheusRegistry            *prometheus.Registry
+	ReportMetadataInterval        time.Duration
+	ServiceBannerRefreshInterval  time.Duration
+	BlockFileTransfer             bool
+	Execer                        agentexec.Execer
+	ContainerLister               agentcontainers.Lister
+	ExperimentalContainersEnabled bool
```

Comment:
Optional: Could we group this option separately (on all structs) so we don't need to reindent the code? The experimental prefix tends to make things long (I did this in #16652 to reduce conflicts).